archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -61,6 +61,17 @@
 public class ArchetypeTest
     extends PlexusTestCase
 {
+    private static final String XML_VERSION_1_0 =
```

Comment:
splitting out these constants makes the tests harder for me to read. I prefer having all the expected content directly in each method, even at the expense of some duplication.

Reply:
Funny how different people read code. I could barley make any sense out of the initial version. Is revert in the new commit.